.github/CODEOWNERS

```diff
@@ -135,6 +135,7 @@
 
 /_maps/ @EOBGames @Maurukas @san7890 @ShizCalev
 /icons/ @Imaginos16 @Krysonism @Twaticus
+/icons/ass @Ghilker
```

Comment:
Multiple owners?

Reply:
maybe someone else want to be asstainer with me